merge_request_approvals.go

```diff
@@ -60,6 +60,26 @@ type MergeRequestApproverGroup struct {
 	}
 }
 
+// MergeRequestApprovalRule represents a GitLab merge request approval rule.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/merge_request_approvals.html#get-merge-request-level-rules
+type MergeRequestApprovalRule struct {
+	ID                   int                                    `json:"id,omitempty"`
+	Name                 string                                 `json:"name,omitempty"`
+	RuleType             string                                 `json:"rule_type,omitempty"`
+	EligibleApprovers    []*BasicUser                           `json:"eligible_approvers,omitempty"`
+	ApprovalsRequired    int                                    `json:"approvals_required,omitempty"`
+	SourceRule           *CreateMergeRequestApprovalRuleOptions `json:"source_rule,omitempty"`
```

Comment:
This should not be an options struct. Those are only used for creating or updating resources.